### PR TITLE
Prepare pages with tables for static rendering

### PR DESCRIPTION
**Pull request by kachkaev**
This PR makes pages like `/dashboard`, `/model` etc. statically renderable. Once we’ve completed the process for all pages, we can deploy the frontend to a static hosting and enable review releases for PR.

### CODE REVIEW

File: src/pages/[visType]/index.tsx [hunk 1]
```diff
@@ -134,4 +135,17 @@ PropagatedPageList.getLayout = function getLayout(page: ReactElement) {
   return <DashboardLayout>{page}</DashboardLayout>;
 };
 
+export const getStaticPaths: GetStaticPaths = async () => {
+  return {
+    paths: ["/dashboard", "/plot", "/analytics", "/model"],
```
Comment (kachkaev) [inline src/pages/[visType]/index.tsx]: @saifulkhan are these all vis types we need? I complied the list by looking at the side menu.

If the number of vis types is static, we are good to go.

If the number of vis types is dynamic but changes rarely, we can make an API request in this function and thus still hardcode the list on `yarn build`.

If the number of vis types is dynamic and changes frequently, we can change the route to something like `list-pages?visType=dashboard` and thus have to build only one server-side route.

**Comment by kachkaev** [comment]
I noticed a small issue with port 3000, it might be to do with the server config. When I try opening the `/dashboard` page while on port 3000, I get:

```
Access to XMLHttpRequest at 'http://vis.scrc.uk/api/v1/template/pages/example/dashboard/'
from origin 'http://localhost:3000' has been blocked by CORS policy: Response to preflight
request doesn't pass access control check: No 'Access-Control-Allow-Origin' header
is present on the requested resource.
```

Running `yarn dev --port 5000` and going to `localhost:5000/dashboard` works though.

File: src/pages/[visType]/index.tsx (hunk 1 above)
Comment (saifulkhan) [reply src/pages/[visType]/index.tsx]: Looks good. Thanks.

**Comment by saifulkhan** [comment]
@kachkaev I allowed the port 3000 to http://vis.scrc.uk. The CORS issue should go away.

**Pull request merged**